src/lib/pipeline_configuration.py

```diff
@@ -74,6 +75,7 @@ class PipelineConfiguration(object):
                    icr_filename="s02e01.csv",
                    run_id_field="rqa_s02e01_run_id",
                    analysis_file_key="rqa_s02e01_",
+                   ws_code=CodeSchemes.WS_CORRECT_DATASET.get_code_with_match_value("s02e01"),
```

Comment:
What is the logic for using the match value as opposed to the string value? The match values aren't in some sense 'canonical'? I could be persuaded either way...

Reply:
string_value was for analysis output. 

We created match_values for this specific, separate purpose of needing to get from strings to codes.